tests/EdFi.Ods.WebApi.IntegrationTests/TestConstants.cs

```diff
@@ -5,5 +5,6 @@
 
 public static class TestConstants
 {
-    public const string BaseUrl = "http://localhost:5449/";
+    public const string SandboxBaseUrl = "http://localhost:5449/";
+    public const string YearSpecificBaseUrl = "http://localhost:5555/";
```

Comment:
Should these be moved to config file?

Reply:
in this case i left them as constants since this is what we did in the other projects. For tests in my opinion its easier to manage.